pages/k8s/upgrade-notes.md

```diff
@@ -207,6 +207,17 @@ juju relate docker kubernetes-worker-docker
 
 <a  id="1.14"> </a>
 
+
+### Clusters that previously ran etcd 2.3
+
+If your cluster has run etcd 2.3 at any point in the past, then it is
+strongly recommended not to upgrade the etcd charm to revision 449. Doing so
+will cause etcd to lose all of its data. For details, see
+https://bugs.launchpad.net/charm-etcd/+bug/1843497
+
+We recommend upgrading etcd directly to charm revision <TBD> instead.
```

Comment:
This will need to be the etcd revision that includes a fix for https://bugs.launchpad.net/charm-etcd/+bug/1843497, most likely the revision for 1.16+CK1

Reply:
Revision added.